siriuspy/siriuspy/diagbeam/bpm/csdev.py

```diff
@@ -247,28 +247,20 @@ def get_switch_database(prefix=''):
                 'type': 'enum', 'enums': Const.SwModes._fields, 'value': 3},
             'SwMode-Sts': {
                 'type': 'enum', 'enums': Const.SwModes._fields, 'value': 3},
-            'SwTagEn-Sel': {
-                'type': 'enum', 'enums': Const.SwTagEnbl._fields, 'value': 0},
-            'SwTagEn-Sts': {
-                'type': 'enum', 'enums': Const.SwTagEnbl._fields, 'value': 0},
-            'SwDataMaskEn-Sel': {
-                'type': 'enum', 'enums': Const.SwDataMaskEnbl._fields,
+            'SwPhaseSyncEn-Sel': {
+                'type': 'enum', 'enums': Const.SwPhaseSyncEnbl._fields,
                 'value': 0},
-            'SwDataMaskEn-Sts': {
-                'type': 'enum', 'enums': Const.SwDataMaskEnbl._fields,
+            'SwPhaseSyncEn-Sts': {
+                'type': 'enum', 'enums': Const.SwPhaseSyncEnbl._fields,
                 'value': 0},
-            'SwDly-SP': {
+            'SwDeswapDly-SP': {
                 'type': 'int', 'value': 0, 'low': 0, 'high': 2**31-1},
-            'SwDly-RB': {
+            'SwDeswapDly-RB': {
                 'type': 'int', 'value': 0, 'low': 0, 'high': 2**31-1},
             'SwDivClk-SP': {
                 'type': 'int', 'value': 0, 'low': 0, 'high': 2**31-1},
             'SwDivClk-RB': {
                 'type': 'int', 'value': 0, 'low': 0, 'high': 2**31-1},
-            'SwDataMaskSamples-SP': {
-                'type': 'int', 'value': 0, 'low': 0, 'high': 2**31-1},
-            'SwDataMaskSamples-RB': {
-                'type': 'int', 'value': 0, 'low': 0, 'high': 2**31-1},
```

Comment:
Why is this removing the PVs which have been changed to `FOFB`?

Reply:
As we talked offline, I did not add new PVs to the database, since they are not being used by any application